packages/editor-ui/src/components/WorkflowCard.vue

```diff
@@ -221,7 +221,7 @@ export default defineComponent({
 				this.$telemetry.track('User opened sharing modal', {
 					workflow_id: this.data.id,
 					user_id_sharer: this.currentUser.id,
-					sub_view: this.$route.name === VIEWS.WORKFLOWS ? 'Workflows listing' : 'Workflow editor',
+					sub_view: 'Workflows listing',
```

Comment:
We will have to circle back to this change, as we'll now have 3 possible places for the sharing modal:
- Home workflow list
- Project specific workflow list
- From within the canvas (while editing a workflow)